docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
```

Comment:
everything in the `docs/` folder is just docusaurus boilerplate - you can ignore. I just included it here to make sur everything works.

I'll be swapping in the content I wrote in the next PR